apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.html.ts

```diff
@@ -0,0 +1,245 @@
+                  the course with students. Things that live across multiple semesters live at the
+                  course level.
+                </p>
+
+                <p>
+                  A <strong>course instance</strong> is the running of an edition of a course that
+                  has assessments, enrollments, grades, etc. Like a semester or quarter.
+                </p>
+
+                <p class="font-italic">
+                  Example: A course might be MATH 101 and have a course instance MATH 101 Fall 2023.
+                </p>
+
+                <p>
+                  For more, see the
+                  <a href="https://prairielearn.readthedocs.io/" target="_blank"
+                    >PrairieLearn User Guide</a
+                  >
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <main class="container mb-4">
+          <h1>Welcome to PrairieLearn</h1>
+
+          <p>
+            To finish the integration for your course, we need to connect
+            <code>${courseName}</code> with a PrairieLearn course instance.
+            <button
+              type="button"
+              class="btn btn-sm btn-info"
+              data-toggle="modal"
+              data-target="#PLterminology"
+            >
+              New here? Learn about our terminology
+            </button>
+          </p>
+
+          <p>
+            If you want this to connect to a <strong>new course</strong> or
+            <strong>new course instance</strong>, you need to create those first and then return to
+            this form.
+          </p>
+          <ul>
+            <li>
+              To create a new course, please <a href="/pl/request_course">request one here</a>.
+            </li>
+            <li>
+              New course instances can be made from the "Course Instances" tab in an existing
+              course.
+            </li>
+            <ul>
+              <li>
+                If you want to copy an existing course instance, you can do that from its Course
+                Instance "Settings" tab.
+              </li>
+            </ul>
+          </ul>
+
+          ${courses.length === 0
+            ? html`<p>
+                It doesn't look like you have any PrairieLearn courses.
+                <a href="/pl/request_course" class="btn btn-success">Go request a course</a>
+              </p>`
+            : html`
+                <p>Your courses:</p>
+                <ul>
+                  ${courses.map((course) => {
+                    return html` <li>
+                      <a href="/pl/course/${course.id}">${course.short_name}: ${course.title}</a>
+                    </li>`;
+                  })}
+                </ul>
+                <form method="POST">
+                  <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
+                  <input type="hidden" name="__action" value="connect_ci" />
```

Comment:
I understand the point, but that's for the `__action` name, which actually isn't used in the post handler since there's only one action from this page. Do we still include an `__action` there? Or omit it when there is only one action?